llama/runner/runner.go

```diff
@@ -288,13 +365,35 @@ func (s *Server) run(ctx context.Context) {
 	}
 }
 
+type Options struct {
+	api.Runner
+
+	NumKeep          int      `json:"n_keep"`
```

Comment:
Note a huge deal, but why re-declare these?

Reply:
The JSON field names are different for communicating to the runner backend (as defined by server.cpp) vs. the Ollama API. I think for now, it is best to just match server.cpp to minimize differences in Ollama while we transition. Once we fully transition we can avoid multiple declarations.

This seemed like the cleanest way to use different JSON fields - is there a better way?

Reply:
Ah that's right. This makes sense then